### DISCUSSION TITLE
language and wikipedia

### DISCUSSION THREAD

**Discussion by RiqqiAmru** [monkeytypegame/monkeytype#4004] [Ideas] [answered]
if country is selected and funbox wikipedia are selected too, can it make it to wikipedia with selected language ?

*in my case with indonesian language but the wikipedia still in english
![image](https://user-images.githubusercontent.com/112304464/219660802-9c50c044-e50b-4b89-a71e-9705f5745767.png)

*sorry about my worst pronoun

**Comment by Miodec** [comment, marked as answer]
At the moment only english is supported
